packages/react/src/radio/Radio.css.ts

```diff
@@ -0,0 +1,70 @@
+import { theme } from "../styles";
+import { style } from "../vanilla-extract";
+
+export const optionWrapper = style({
```

Comment:
can be simplified to just option
```suggestion
export const option = style({
```